main.css

```diff
@@ -0,0 +1,63 @@
+table {
+  text-align: center;
+  width: 450px;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+}
+#main {
+  font-family: Ubuntu, sans-serif;
+  text-align: center;
+  width:500px;
+  border: 2px solid black;
+  padding: 10px;
+  height: auto;
+  margin-top: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 30px;
```

Comment:
You're starting to use CSS pretty well. I'd now look at how you could use Sass to extract some of these magic numbers to meaningful variable names. You'll also find yourself writing less tricky CSS that way.